config.md

```diff
@@ -7,304 +7,170 @@ This specification outlines the JSON format describing images for use with a con
+
+- **os** *string*, REQUIRED
+
+  The name of the operating system which the image is built to run on.
+  Possible values include: `darwin`, `freebsd`, `linux`.
+  More values may be supported in the future and any of these may or may not be supported by a given container runtime implementation.
+  New entries SHOULD be submitted to this specification for standardization and be inspired by the [Go language documentation for $GOOS and $GOARCH](https://golang.org/doc/install/source#environment).
+
+- **config** *object*, OPTIONAL
+
+  The execution parameters which should be used as a base when running a container using the image.
+  This field can be <code>null</code>, in which case any execution parameters should be specified at creation of the container.
+
+   - **user** *string*, OPTIONAL
+
+     The username or UID which the process in the container should run as.
+     This acts as a default value to use when the value is not specified when creating a container.
+     All of the following are valid: `user`, `uid`, `user:group`, `uid:gid`, `uid:group`, `uiser:gid`
+     If `group`/`gid` is not specified, the default group and supplementary groups of the given `user`/`uid` in `/etc/passwd` from the container are applied.
+
+   - **Memory** *integer*, OPTIONAL
+
+     Memory limit (in bytes).
+     This acts as a default value to use when the value is not specified when creating a container.
+
+   - **MemorySwap** *integer*, OPTIONAL
+
+     Total memory usage (memory + swap); set to <code>-1</code> to disable swap.
+     This acts as a default value to use when the value is not specified when creating a container.
+
+   - **CpuShares** *integer*, OPTIONAL
+
+     CPU shares (relative weight vs. other containers).
+     This acts as a default value to use when the value is not specified when creating a container.
+
+   - **ExposedPorts** *object*, OPTIONAL
+
+     A set of ports to expose from a container running this image.
+     Its keys can be in the format of:
+`port/tcp`, `port/udp`, `port` with the default protocol being `tcp` if not specified.
+     These values act as defaults and are merged with any specified when creating a container.
+     **NOTE:** This JSON structure value is unusual because it is a direct JSON serialization of the Go type <code>map[string]struct{}</code> and is represented in JSON as an object mapping its keys to an empty object.
+
+   - **Env** *array of strings*, OPTIONAL
+
+     Entries are in the format of <code>VARNAME="var value"</code>.
+     These values act as defaults and are merged with any specified when creating a container.
+
+   - **Entrypoint** *array of strings*
+
+     A list of arguments to use as the command to execute when the container starts.
+     This value acts as a  default and is replaced by an entrypoint specified when creating a container. This field MAY be "null".
```

Comment:
Also remove the doubled space: “as a  default“ → “as a default”.  And move “This field MAY be `null`” to its own line.